docs/poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.2 and should not be changed by hand.
```

Comment:
Same comment as the last time: https://github.com/scylladb/scylla-operator/pull/2585#discussion_r2037191214.
For future reference, it's probably the easiest to follow this https://github.com/scylladb/scylla-operator/blob/master/docs/README-dev.md#update-dependencies.